packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -538,6 +570,21 @@ export default class Health extends Component {
 
   _getSrUrl = sr => `srs/${sr.id}`
 
+  _getDuplicatedMacAddress = createSelector(
+    () => this.props.vifsByMac,
+    macAddresses => {
+      const vifsWithDuplicatedMacAddresses = {}
+      let i = 0
+      forEach(macAddresses, macAddress => {
+        if (macAddress.length > 1) {
+          vifsWithDuplicatedMacAddresses[Object.keys(macAddresses)[i]] = macAddress
+        }
+        i++
+      })
+      return vifsWithDuplicatedMacAddresses
+    }
+  )
+
```

Comment:
Now, we need the duplicated MAC adresses list and you can pass `vifsByMac` to show VIFs/VMs/networks [`data-vifsByMac`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/sorted-table/index.js#L285).